orttraining/orttraining/python/training/onnxblock/onnxblock.py

```diff
@@ -187,7 +187,30 @@ def __call__(self, *args, **kwargs):
 
         output = self.build(*args, **kwargs)
 
-        model = onnx.shape_inference.infer_shapes(accessor._GLOBAL_ACCESSOR.model)
+        model = accessor._GLOBAL_ACCESSOR.model
+
+        size_check = False
```

Comment:
Can we rename this variable to something like `model_exceeds_max_permissible_size`?